app/models/concerns/redis_form.rb

```diff
@@ -50,7 +50,9 @@ def save
 
   def form_matches_schema
     if form.present?
-      errors[:form].concat(JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form))
+      JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form).each do |v|
```

Comment:
Each still returns an iterator instead of the original object.

```suggestion
      JSON::Validator.fully_validate(VetsJsonSchema::SCHEMAS[self.class::FORM_ID], parsed_form).tap do |v|
```